_posts/patterns/2015-04-15-icons.md

```diff
@@ -547,7 +558,7 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Sort
+            sort
         </div>
     </div>
     <div class="example example-icon">
```

Comment:
Unfortunately, I can't comment on the exact line.  It looks like "Undo" (line 572) should be lowercase and also include "rotate-left" as a synonym (or "rotate-left" should be removed from the synonym list).